portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path

```diff
@@ -990,6 +990,26 @@
 	<td>//li[contains(@class,'nav-item')]//button[contains(@title,'${key_webContentTitle}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>GRAPHQL</td>
+	<td>//button[contains(.,'GraphQL')]</td>
+	<td></td>
```

Comment:
maybe we could join the GRAPHQL, HEADERS and ADD_HEADER by creating a generic locator with {value} to pass? And we would be passing the value in the macro or even in the step, example: `//button[contains(.,'${action}')]`

Reply:
Great, can't agree more, I saw there are many //button[...contains...] in Button.path file, so I didn't combine these at the first time, will update :dancers: